src/transformers/models/distilbert/modeling_distilbert.py

```diff
@@ -141,7 +142,10 @@ def __init__(self, config: PretrainedConfig):
         self.dim = config.dim
         self.dropout = nn.Dropout(p=config.attention_dropout)
 
-        assert self.dim % self.n_heads == 0
+        # Have an even number of multi heads that divide the dimensions
+        if self.dim % self.n_heads != 0:
+            # Raise value errors for even multi-head attention nodes
+            raise ValueError(f"self.n_heads: {self.n_heads} must divide self.dim: {str(self.dim)} evenly")
```

Comment:
```suggestion
            raise ValueError(f"self.n_heads: {self.n_heads} must divide self.dim: {self.dim} evenly")
```

Reply:
Hello @younesbelkada, may I suggest to change the comment on line 147 with "Raise ValueErrors for **uneven** multi-head attention nodes"